gcm_filters/kernels.py

```diff
@@ -132,8 +136,8 @@ def __call__(self, field: ArrayType):
             (out - np.roll(out, -1, axis=-2)) / self.dys * self.dxs
         )  # flux across southern cell edge
 
-        wflux = wflux * self.w_wet_mask  # no-flux boundary condition
-        sflux = sflux * self.s_wet_mask  # no-flux boundary condition
+        wflux = self.kappa_w * wflux * self.w_wet_mask  # no-flux boundary condition
+        sflux = self.kappa_s * sflux * self.s_wet_mask  # no-flux boundary condition
```

Comment:
I think we could make the filter faster if we multiply `self.w_wet_mask` and `self.s_wet_mask` by their respective kappas in the post_init method, to avoid re-computation in every Laplacian iteration. What do you think?

Reply:
Yes, I agree. I talked about doing this during our last meeting, but I completely forgot to change that. I will make this change.